src/styles/theme/themes/dark.ts

```diff
@@ -46,6 +46,7 @@ const darkTheme = {
     shadow: colors.black,
     componentBG: colors.productDark100,
     hoverComponentBG: colors.productDark300,
+    messageHighlightBG: colors.messageHightlightDark,
```

Comment:
```suggestion
    messageHighlightBG: colors.messageHighlightDark,
```
Corrected variable name should be used here

Reply:
cc: @nkdengineer